AxonDeepSeg/models/default_SEM_model_v1/readme.txt

```diff
@@ -0,0 +1,7 @@
+Current training/validation/testing dataset:
+
+ Patches of 512x512 pixels
+ 3 classes (axon/myelin/background)
+ Rat microscopy
+ white matter spinal cord
+Path to dataset: /Volumes/projects/axondeepseg/labelled_datasets/SEM_3c_512
```

Comment:
That path is where you mounted it on your computer; you should instead specify that it's on `duke:` instead of your own `/Volumes` folder

Reply:
Resolved in commit d0e33c5.